### ISSUE TITLE
Data - image upload - upload results screen

### ISSUE THREAD

**Issue by anadm** [SpeciesFileGroup/taxonworks#708]
When I go to images/new, I click on the upload box and select N files to be uploaded, then "OK". 
TW shows a progress bar for each picture. However, no acknowledgement of "done" appears once upload is finished. 
This causes some strangeness on the user end until you realize you have to click in each image etc. 
Alternatively (see https://github.com/SpeciesFileGroup/taxonworks/issues/692 and https://github.com/SpeciesFileGroup/taxonworks/issues/693):
select file 
  upload file
   progress bar appears
    done message appears
     list of files uploaded appears, prompting to fill out short description (say, 50 chars max) (short description would become a requirement so that file can be properly displayed in human-friendly list in /images/list)
![image](https://user-images.githubusercontent.com/1326839/49513897-920f2f00-f879-11e8-8618-0a354563eaee.png)

Example short descriptions: 
"Exemplus exemplus BMNH(E)00234897 head frontal" 
"Exemplus exemplus habitus" (in this case good practices would expect user to associate image with specimen)
"screen capture of hub with numbered areas"

**Comment by mjy** [comment]
@anadm Does the use of the data annotator now permit what you want to do?  For example try the radial in the show window?  If so, we can include the radial in the list, and beside the drag/drop image form.

**Comment by anadm** [comment]
No, I'd much expect to see a text description in the table and have the fingerprint in the radial annotator if that's the case

**Comment by mjy** [comment]
What I mean is- does the radial annotator contain enough of the types of metadata to add what you need?  I.e. displaying them in a list, or browse/gallery functionality can include the fields, we just want to make sure you can record the initial bits.

It's very difficult to think about what to require beyond the image, since we can use images in many different ways within a research context, so I'm hesistant to require something, but certainly we can through some red flags up ('there is no data associated with this image').  Moving forward we expect most images to ulimately be added in context that has metadata inherently with them (e.g. dropping a depiction on a specimen), rather than using the default data.

**Comment by anadm** [comment]
I think the annotator may allow a user to add tags, but still the list should display a short description, and some basic description should be a requirement.
